decryptor/base/utils.go

```diff
@@ -14,13 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package mysql
+package base
```

Comment:
it's mysql related functions. they should be stored in `mysql` package, not in `base`)